### DISCUSSION TITLE
If there was a map with Aim Assist in the playlist, then it broke after the last update

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#17736] [Q&A] [answered]
Several errors arose due to the latest update, which changed the name of the Aim Assist mod.
1. Mod icon turned gray and shows as "unknown mod"
2. The map no longer runs in this playlist
3. The leaderboard has been reset.

The playlist is called "osu!content playlist #3" and will be active until April 29th.
![osu_2022-04-09_10-49-13](https://user-images.githubusercontent.com/59089206/162562409-0e698260-e82a-42d6-86bb-1dfc3d5d7b1c.jpg)
![osu_2022-04-09_10-49-24](https://user-images.githubusercontent.com/59089206/162562412-09bad093-9962-4d7f-91ce-4c87a7da1d0b.jpg)
![osu_2022-04-09_10-49-43](https://user-images.githubusercontent.com/59089206/162562415-992f9d10-ad86-46fb-a028-93e4a45e4ca9.jpg)

[performance.log](https://github.com/ppy/osu/files/8456468/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8456469/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8456470/updater.log)
[database.log](https://github.com/ppy/osu/files/8456471/database.log)
[network.log](https://github.com/ppy/osu/files/8456472/network.log)

**Comment by peppy** [comment, marked as answer]
The best I can offer is manually fixing the playlist for you. Does that sound okay?

**Reply by Moltanicaa** [reply]
As far as I know, it's not possible to edit playlists right now. If it could be done that way, it would be ok. But the fact that the leaderboard is reset is unpleasant.

**Reply by peppy** [reply]
no leaderboard would be reset

**Reply by peppy** [reply]
I've applied a fix.
